src/gravel/controllers/orch/ceph.py

```diff
@@ -150,13 +150,14 @@ def fsid(self) -> str:
             raise CephError(str(e))
 
     def _cmd(self, func: Callable[[str, bytes], Any],
-             cmd: Dict[str, Any]
+             cmd: Dict[str, Any],
+             inbuf: bytes = b""
              ) -> Any:
         self.assert_is_ready()
         cmdstr: str = json.dumps(cmd)
 
         try:
-            rc, out, outstr = func(cmdstr, b"")
+            rc, out, outstr = func(cmdstr, inbuf)
```

Comment:
```suggestion
            rc, out, outstr = func(cmdstr, inbuf.encode("utf-8'))
```

might be cleaner todo the bytes conversion here ?

Reply:
Personally, I think it's better to make it clear that these functions are taking `bytes` instead of string, and do the conversion as earlier in the call stack as possible. That matches the expectation of an actual `call()` to the underlying thing.